app/api/routes/image_queries.py

```diff
@@ -224,18 +160,26 @@ async def post_image_query(  # noqa: PLR0913, PLR0915, PLR0912
                 rois=results["rois"],
                 text=results["text"],
             )
-            app_state.db_manager.create_iqe_record(iq=image_query)
+            background_tasks.add_task(app_state.db_manager.create_iqe_record, iq=image_query)
```

Comment:
Moving the db transaction to after the response is returned.
is this even worth it? need to benchmark.